docs/how-to/hip_porting_driver_api.rst

```diff
@@ -1,33 +1,30 @@
 .. meta::
   :description: This chapter presents how to port the CUDA driver API and showcases equivalent operations in HIP.
-  :keywords: AMD, ROCm, HIP, CUDA, driver API
+  :keywords: AMD, ROCm, HIP, CUDA, driver API, porting, port
 
 .. _porting_driver_api:
 
 *******************************************************************************
 Porting CUDA driver API
 *******************************************************************************
 
-NVIDIA provides separate CUDA driver and runtime APIs. The two APIs have
-significant overlap in functionality:
-
-* Both APIs support events, streams, memory management, memory copy, and error
-  handling.
-
-* Both APIs deliver similar performance.
+CUDA provides separate driver and runtime APIs. The two APIs generally provide
+the same functionality, however the driver API allows for more fine-grained
+control over initialization and context- and module-management. This is all
+taken care of implicitly by the runtime API.
 
 * Driver API calls begin with the prefix ``cu``, while runtime API calls begin
   with the prefix ``cuda``. For example, the driver API contains
   ``cuEventCreate``, while the runtime API contains ``cudaEventCreate``, which
   has similar functionality.
 
-* The driver API defines a different, but largely overlapping, error code space
-  than the runtime API and uses a different coding convention. For example, the
-  driver API defines ``CUDA_ERROR_INVALID_VALUE``, while the runtime API defines
-  ``cudaErrorInvalidValue``.
+* The driver API offers two additional functionalities not directly provided by
+  the runtime API: ``cuModule`` and ``cuCtx`` APIs.
```

Comment:
"The driver API offers two additional functionalities not directly provided by
  the runtime API: ``cuModule`` and ``cuCtx`` APIs."
Should be:
The driver API offers two additional low-level functionalities not exposed by the runtime API: module management ``cuModule*`` and context management ``cuCtx*`` APIs.

Reply:
Fixed